spec/services/usps_in_person_proofing/proofer_spec.rb

```diff
@@ -201,6 +219,30 @@ def expect_facility_fields_to_be_present(facility)
       ).to eq(1)
     end
 
+    context 'when the user is going through EIPP' do
+      let(:usps_eipp_sponsor_id) { '314159265359' }
+      let(:ipp_assurance_level) { '2.0' }
+      let(:is_enhanced_ipp) { true }
+      before do
+        allow(IdentityConfig.store).to receive(:usps_eipp_sponsor_id).
+          and_return(usps_eipp_sponsor_id)
+      end
+      it 'uses the EIPP usps_ipp_sponsor_id and ipp_assurance_level in calls to the USPS API' do
+        stub_request_eipp_facilities
+        subject.request_facilities(location, is_enhanced_ipp)
+
+        expect(WebMock).to have_requested(:post, request_url).
+          with(
+            body: hash_including(
+              {
+                sponsorID: usps_eipp_sponsor_id.to_i,
+                IPPAssuranceLevel: ipp_assurance_level,
```

Comment:
@eileen-nava  I am seeing `usps_eipp_client_id` used above (config/application.yml.default) and `usps_eipp_sponsor_id` used  like so (IdentityConfig.store.usps_eipp_sponsor_id).  Wondering if it should be the same name? Sorry if I goofed but have a look and let me know what you think.